h/models/group.py

```diff
@@ -126,6 +126,13 @@ def type(self):
     def is_public(self):
         return self.readable_by == ReadableBy.world
 
+    @property
+    def logo(self):
+        """Returns logo icon if one exists for this groups authority else
+        returns None."""
+        return {'biopub.hypothes.is': 'biopub-logo',
+                'hypothes.is': 'logo'}.get(self.authority)
```

Comment:
In development environments the default authority name is `localhost`, but we probably do want to use the regular Hypothesis logo in that case? Could we simply make `'logo'` the default?

Reply:
It's possible to have a group that doesn't have an assigned logo and in that case we don't want to display the logo at all, so I don't want to make it the default meaning that if one isn't assigned default to hypothesis's logo.  I do think it's reasonable to say whatever the default authority is, make it have the hypothesis logo.